website-frontend/src/routes/academics/[slug]/+page.svelte

```diff
@@ -0,0 +1,51 @@
+        { slug: "ms-bioinformatics", title: "Master of Science in Bioinformatics", image: "https://placehold.co/600x400", description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam vel ullamcorper nulla. Morbi nec urna ac tellus tempor blandit id id dui. Donec hendrerit sem a nibh lobortis placerat. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut erat massa, luctus quis arcu non, posuere vulputate lacus. Nulla euismod quam eget dolor tristique, pulvinar egestas leo imperdiet. Curabitur erat augue, placerat non turpis non, posuere rutrum est. Nullam scelerisque quam et nisl blandit molestie. Vestibulum at velit id dui faucibus pretium. Cras aliquet ante at scelerisque eleifend.<br><br>Sed non odio scelerisque, iaculis ante ac, viverra mauris. Maecenas nibh elit, dignissim at felis sed, ullamcorper pretium enim. Duis at viverra dolor. Praesent vestibulum mollis malesuada. Fusce sit amet ante aliquet, pellentesque nibh eget, consequat magna. Nullam et ipsum non nulla mollis iaculis. Etiam non tempus risus.", curriculum: "" },
+        { slug: "phd-cs", title: "Doctor of Philosophy in Computer Science", image: "https://placehold.co/600x400", description: "The program aims to develop computer scientists who are armed with methods, tools and techniques from both theoretical and systems aspects of computing. They should be able to formulate computing problems and develop new and innovative technology as novel solutions to address those problems. The graduates will gain expertise to independently contribute in Research and Development (R&D) on a specialized area of Computer Science. The program will prepare graduates for professional and/or research careers in industry, government or academe.", curriculum: "" },
+    ];
+
+    const currentProgram = derived(page, $page => {
+        const slug = $page.params.slug;
```

Comment:
This is _not_ the way we do router parametrization in Svelte. Refer to this page for help on [Routing in Svelte](https://svelte.dev/docs/kit/routing#error). Usually you'd need to make a `+page.server.ts` where you have a `load` function that checks its `{ params }` object. This object has a `params.slug` parameter which we can match against _and then it loads the content dependent on the slug._